genesyscloud/architect_flow/resource_genesyscloud_architect_flow_utils.go

```diff
@@ -84,7 +100,21 @@ func architectFlowResolver(flowId, exportDirectory, subDirectory string, configM
 
 	log.Printf("Updating resource config and state file for flow '%s'", flowId)
 	updateResourceConfigAndState(configMap, resource, exportDirectory, subDirectory, filename)
-	return nil
+	return err
+}
+
+// sanitizeFlowName will replace all forward slashes, backslashes and white spaces with an underscore
+func sanitizeFlowName(s string) string {
+	// First replace empty strings (multiple spaces) with a single underscore
```

Comment:
How about just replacing all non-alphanumeric characters with underscore?

Reply:
I was going to go with that, but I thought leaving special characters in will make it easier to identify which flows are in which files. If a flow name is mostly special characters, it will come out like `___Foo_1___2___-{ID}.yaml`. I could see that being a bit of a inconvenience for users